index.bs

```diff
@@ -1516,6 +1527,8 @@ When this operation is invoked, the authenticator must perform the following pro
     so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
 - If the |requireResidentKey| flag is set to |true| and the authenticator cannot store a [=Client-side-resident Credential
     Private Key=], return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
+- If the |requireRateLimiting| flag is set to |true| and the authenticator cannot support [=Rate Limiting=], return an error code 
```

Comment:
@gmandyam I do not think the webauthn specification is the proper place for stating detailed authenticator operational characteristics and requirements. One obvious option is to point to appropriate FIDO Alliance documents and programs, such as authenticator certification.